src/app/legacy/containers/PodcastPromo/Inline.jsx

```diff
@@ -33,35 +39,31 @@ const GEL_GROUP_1_WIDTH_360PX = '22.5rem';
 
 const ResponsivePodcastPromoWrapper = styled.div`
   ${({ dir }) => (dir === 'ltr' ? 'float: right;' : 'float: left;')}
-  background: ${props => props.theme.palette.LUNAR};
-  margin: ${GEL_SPACING_TRPL} 0;
+
+  margin: 0rem ${GEL_SPACING} ${GEL_SPACING_TRPL} ${GEL_SPACING};
```

Comment:
I think we could use shorthand here: 0rem ${GEL_SPACING} ${GEL_SPACING_TRPL} 

See the section here: https://www.w3schools.com/css/css_margin.asp
If the margin property has three values: 
margin: 25px 50px 75px;
top margin is 25px
right and left margins are 50px
bottom margin is 75px